androidapp/app/src/main/java/com/example/lanweather/MainActivity.kt

```diff
@@ -22,5 +33,42 @@ class MainActivity : AppCompatActivity() {
             R.id.navigation_home, R.id.navigation_forecast, R.id.navigation_settings))
 //        setupActionBarWithNavController(navController, appBarConfiguration)
         navView.setupWithNavController(navController)
+
+        // Only used to test notifications
+//        val button: Button = findViewById(R.id.test_notification)
+//        button.setOnClickListener { displayNotification() }
+    }
+
+    private fun displayNotification() {
+        val builder = NotificationCompat.Builder(this, CHANNEL_ID)
+            .setSmallIcon(R.drawable.ic_cloud_black_24dp)
+            .setContentTitle(getString(R.string.clothing))
+            .setContentText(String.format(getString(R.string.wear_this_insert),
+                getString(R.string.jacket)))
```

Comment:
So this would be changed to send appropriate suggestions based on current weather?

Reply:
Yeah that's exactly what would happen.